syntax = "proto3";
package rdma;

message MemoryResourceRequest {
    uint64 size = 1;
    uint64 offset = 2;
    uint32 type = 3;
    string name = 4;
    bool persistent = 5;
}

